src/Prowlarr.Http/ClientSchema/SchemaBuilder.cs

```diff
@@ -28,13 +38,20 @@ public static List<Field> ToSchema(object model)
                 var field = mapping.Field.Clone();
                 field.Value = mapping.GetterFunc(model);
 
+                if (_configFileProvider.AuthenticationMethod is AuthenticationType.None or AuthenticationType.External &&
```

Comment:
```suggestion
                if (_configFileProvider.AuthenticationMethod is AuthenticationType.None &&
```

punishing users of what Servarr Team pushes to be more secure than built in auth doesn't make sense (I.e. nginx auth, authelia, etc.)

Reply:
Yea, if I understand this, we are just allowing passwords open for those with auth explicitly enabled in app... vs all the other arr's where passwords are never shown (auth or not). This is actually opening things a bit from Sonarr/Radarr logic, not punishing users with External. I'm fine with this approach.